File: Maratona/Maratona 2022/soluções/G - NumeroEuller/NumeroEuller.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class NumeroEuller {

    public static void main(String[] args) throws IOException {

        String linha = "";
        double n;
        double s;

        BufferedReader entrada;

        entrada = new BufferedReader(new InputStreamReader(System.in));

        while ((linha = entrada.readLine()) != null) {

            if (linha.equals("0")) {
                break;
            }

            n = Double.parseDouble(linha);
            s = 1 + (1 / n);
            s = Math.pow(s, n);

            System.out.printf("%.24f %n", s);
        }
        System.exit(0);
    }
}

File: Maratona/Maratona 2022/soluções/G - NumeroEuller/NumeroEuller.py
import sys
if __name__ == '__main__':
    while True:
        linha = input()
        if linha in '0':
            break
        n = int(linha)
        print(f'{(1+(1/n)):,.24f}')
        s = (1+(1/n))**n
        print(f'{s:,.24f}')
sys.exit()
